Source/WarpX.cpp

```diff
@@ -642,6 +643,8 @@ WarpX::ReadParameters ()
         {
             getWithParser(pp_warpx, "do_multi_J_n_depositions", do_multi_J_n_depositions);
         }
+        pp_warpx.query("do_asymmetrical_psatd", do_asymmetrical_psatd);
```

Comment:
Could we read this as part of the `psatd.` input parameters? We could maybe call it `psatd.do_asymmetrical` or something similar.

Reply:
Changed to more general name `do_asymmetrical`, which is important since we are planning to implement along with asymmetrical multiJ also asymmetrical PSATD and asymmetrical Galilean.